docs/nanocloud-api.yml

```diff
@@ -101,25 +101,35 @@ paths:
           required: false
       responses:
         200:
-          description: Fetch all users informations without "me" parameter,
-            fetch current user informations with it
+          description: If you're admin, Fetch all users informations, otherwise, if you add "me" parameter, fetch current user informations
```

Comment:
If you're admin, Fetch all users informations, otherwise, if you add "me" parameter, fetch current user informations -> As an admin, get all users or current user if "me" parameter is provided.